utils/unittest/fixtures.go

```diff
@@ -777,11 +802,13 @@ func ServiceEventsFixture(n int) flow.ServiceEventList {
 	sel := make(flow.ServiceEventList, n)
 
 	for ; n > 0; n-- {
```

Comment:
could you reverse the loop iteration?  `for i := 0; i < n; i++` seems more neutral